client/src/file/mod.rs

```diff
@@ -95,6 +95,36 @@ impl Keyring {
     /// * `path` - The path to the file backend.
     /// * `secret` - The service key, usually retrieved from the Secrets portal.
     pub async fn load(path: impl AsRef<Path>, secret: Secret) -> Result<Self, Error> {
+        Self::load_inner(path, secret, true).await
+    }
+
+    /// Load from a keyring file.
+    ///
+    /// # Arguments
+    ///
+    /// * `path` - The path to the file backend.
+    /// * `secret` - The service key, usually retrieved from the Secrets portal.
+    ///
+    /// # Safety
+    ///
+    /// The secret is not validated to be the correct one to decrypt the keyring
+    /// items. Allowing the API user to write new items with a different
+    /// secret on top of previously added items with a different secret.
+    ///
+    /// As it is not a supported behaviour, this API is mostly meant for
+    /// recovering broken keyrings.
+    pub async unsafe fn load_unchecked(
```

Comment:
Could we also have a `load_default_unchecked` maybe, for simplicity?

Reply:
Well that's the case that we have. I'm not sure why. Maybe the user migrated their keyring to a new system so they got a different secret, but it was not catched at the time?

Reply:
No idea how did any of that happen.

Reply:
The problem is that i also have a user here https://gitlab.gnome.org/World/Authenticator/-/issues/452 reporting a similar one. Although, I don't know how many broken items they had. 

The thing about making it always fail if there is any  invalid item makes the load API not work around those items which seemed like good enough for me, imho.

Reply:
My plan was to add a one-time migration in Fractal that removes broken items automatically, so we don't have to worry about it after upgrading oo7. But if it reoccurs after that, it's clearly a problem.

That said if you want to keep the behavior you had, it's also fine by me. There's a probability that users will open an issue because they see the same error every time that they open the app though. Or maybe just put it as a warning, because it doesn't actually block the application.

Reply:
I have switched the behaviour back to the one I had. We can see what the users end up reporting with their logs and figure out a plan from there instead of over-doing it now.

If this looks lgtm, I will merge and release it as 0.4.2